app/models/services/index.ts

```diff
@@ -120,6 +120,25 @@ export function buildService(
   return new ServiceClass(serviceConfig, operation);
 }
 
+/**
+ *  Returns any services that support concatenation from the list of configs
+ * @param configs - The potential matching service configurations
+ * @returns any configurations that support concatenation
+ */
+function supportsConcatenation(configs: ServiceConfig<unknown>[]): ServiceConfig<unknown>[] {
+  return configs.filter((config) => getIn(config, 'capabilities.concatenation', false));
+}
+
+/**
+ * Returns any services that don't support concatenation from the list of configs
+ * @param configs - The potential matching service configurations
+ * @returns any configurations that don't support concatenation
+ */
+function supportsOperationsOtherThanConcatenation(
+  configs: ServiceConfig<unknown>[]): ServiceConfig<unknown>[] {
+  return configs.filter((config) => !getIn(config, 'capabilities.concatenation', false));
```

Comment:
Oh reading this logic I see the issue. The intent is that we should not send a request to a service chain that requires concatenation. That should not be based on the `capabilities`. If we want to revert this change and write a separate ticket I'd be good with that.